### PR TITLE
Added SplashScreen

### PR DESCRIPTION
**Pull request by corybrooks**
Added 1.5 second SplashScreen that occurs before LoginActivity and MainActivity. 

To implement this, the android manifest has been adjusted so that the SplashScreen activity has the intent filter of the main activity, and the SplashScreen class itself calls the Login Activity. 
This new flow ensures that the same activity order from before is maintained, but the splashScreen occurs first.

### CODE REVIEW

File: android/app/src/main/java/com/google/codeu/chatme/view/SplashScreen.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+public class SplashScreen extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_splash_screen);
+
+        final TextView txtView = (TextView) findViewById(R.id.textView);
+        txtView.startAnimation(AnimationUtils.loadAnimation(SplashScreen.this, android.R.anim.slide_in_left));
```
Comment (blong1996) [inline android/app/src/main/java/com/google/codeu/chatme/view/SplashScreen.java:21]: nice implementation of the animation!

**Review by blong1996 — APPROVED** [review]
Love the Splash Screen bro! Are you good with graphic design software or making logos? It would be really nice to create a logo to replace "Chat Me" on the splash screen and the login screen. Also, an App Icon would pair well with this too!

File: android/app/src/main/java/com/google/codeu/chatme/view/SplashScreen.java (hunk 1 above)
Comment (ymittal) [reply android/app/src/main/java/com/google/codeu/chatme/view/SplashScreen.java:21]: Looks cool, @corybrooks!

File: android/app/src/main/java/com/google/codeu/chatme/view/SplashScreen.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+        Handler handler = new Handler();
+        handler.postDelayed(new Runnable() {
+            @Override
+            public void run() {
+                startActivity(new Intent(SplashScreen.this, LoginActivity.class));
+                finish();
+            }
+        }, 1500);
```
Comment (ymittal) [inline android/app/src/main/java/com/google/codeu/chatme/view/SplashScreen.java]: Please extract magic number as a constant (`DELAY_IN_MILLISECONDS`?)

File: android/app/src/main/java/com/google/codeu/chatme/view/SplashScreen.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+
```
Comment (ymittal) [inline android/app/src/main/java/com/google/codeu/chatme/view/SplashScreen.java]: Remove extra lines

File: android/app/src/main/java/com/google/codeu/chatme/view/MainActivity.java [hunk 4]
```diff
@@ -14,6 +14,6 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
 
         // TODO: implement basic splash screen in this activity
-        startActivity(new Intent(this, LoginActivity.class));
+        startActivity(new Intent(this, SplashScreen.class));
```
Comment (ymittal) [inline android/app/src/main/java/com/google/codeu/chatme/view/MainActivity.java]: I think you can just delete `MainActivity` and its layout now. It was just a placeholder for the `SplashScreen` activity you created.

**Review by ymittal — COMMENTED** [review]
Really cool stuff, @corybrooks! Splash Screen is clean. Please merge `dev` into `SplashScreen` (conflicts arose after the latest PR got merged) and look at my suggestions. We should be good to move forward then.

**Pull request merged**